tutorials/arvo/claz.md

```diff
@@ -0,0 +1,76 @@
+
+`%mainnet` specifies which network this is going to sent out on.
+
+`author` here is an etherium address that is issuing the transaction. Remember that it will need to be written all in lower case, dotted notation, e.g.  0x3b17.d097.d9dd.711e.4ef8.517a.bbf1.8b2b.a643.81fe
+
+`%single` specifies what kind of batch that will be created. If you are going to do multiple transactions you will need to either set this to `%more` and create a list of batches or upload them before generating the next one. The reason this ship needs to have internet access is because it needs to look up the nonce for the specified address.
+
+`%spawn` is the command we want to issue. After this what the arguments are will vary based on the command.
+
+`~ship` will be the ship to be spawned.
+
+`to` is the etherium address to spawn the ship to.
+
+You will want to read `sur/claz.hoon` which you can find [here](https://github.com/urbit/urbit/blob/master/pkg/arvo/sur/claz.hoon).
+This will contain more details about the various kinds of transactions you can perform particularly `++call`
+
+After running this command you should find a csv file at the specified path. You can examine this file to learn how they are constructed but it's not important for this guide.
+
+This file can now be signed and uploaded. At this point, you can transfer this file to an airgapped machine which has your keys if you want. If you choose to do that you will need to download the urbit binary as well as the most recent pill file or a copy of the fakezod pier you have already booted.
+
+For this guide, we'll continue on the same ship.
+
+Run `|start %eth-sender` and then you can use it to sign the transaction file.
+
+```
+:eth-sender [%sign %/transaction-signed/txt %/transaction/eth-txs %/mykey/txt ~[80]]
+```
+
+Let's again break this down.
+
+`%sign` is the action we are going to perform here.
+
+`%/transaction-signed/txt` is the path to the output file.
+
+`%/transaction/eth-txs` is the path to the transaction file we generated earlier. It will need to be accessible so don't forget to `|comit %home` if you moved it to a different pier.
+
+`%/mykey/txt` is the path to your private key file. This file is most likely the reason you will want to use an airgapped machine so that your private key never touches a networked computer. Opsec is left to the reader.
+
+`~[80]` is a list of possible gwei costs a signed transaction file will be created for each one. Here we just specify 80 gwei. You may need to adjust this based on current etherium network traffic and average gas costs.
+
+Now you should have a directory in your pier that contains one file for each of the specified gwei costs.
+
+To actually transmit the transaction, paste the contents of one of these files into https://etherscan.io/pushTx or whatever other transmission system you would like to use. **This is the action that will actually cost you etherium. Only do this step if you want to actually perform the action.**
```

Comment:
Same comment about what risk the emphasize as above. Transaction costs are trivial compared to, well, even planet value I guess.